app/models/reducers/count_reducer.rb

```diff
@@ -1,10 +1,24 @@
+# Reducers::CountReducer
+#
+# data: {
+#  extracts: INT,
+#  classifications: INT
+# }
+#
 module Reducers
   class CountReducer < Reducer
-    def reduction_data_for(extracts)
+    def reduction_data_for(extracts, reduction)
+      data = {}
+      data = reduction.data if running_reduction? && reduction&.data.present?
```

Comment:
What if we removed this, and inside `reduction_data_for` it's always a running_reduction, but it might start with an empty data/store.